Makefile

```diff
@@ -148,17 +148,27 @@ CONTROLLER_DEEPCOPY := api/v1alpha1/zz_generated.deepcopy.go
 $(CONTROLLER_DEEPCOPY): | $(CONTROLLER_GEN)
 	$(CONTROLLER_GEN) object:headerFile="$(CURDIR)/hack/boilerplate.go.txt,year=$(shell date +"%Y")" paths="./..."
 
-# Generate embedded YAMLs
-EMBEDDED_YAMLS := pkg/embeddedyamls/yamls.go
-$(EMBEDDED_YAMLS): pkg/embeddedyamls/generators/yamls2go.go deploy/crds/submariner.io_servicediscoveries.yaml deploy/crds/submariner.io_brokers.yaml deploy/crds/submariner.io_submariners.yaml deploy/submariner/crds/submariner.io_clusterglobalegressips.yaml deploy/submariner/crds/submariner.io_clusters.yaml deploy/submariner/crds/submariner.io_endpoints.yaml deploy/submariner/crds/submariner.io_gatewayroutes.yaml deploy/submariner/crds/submariner.io_gateways.yaml deploy/submariner/crds/submariner.io_globalegressips.yaml deploy/submariner/crds/submariner.io_globalingressips.yaml deploy/submariner/crds/submariner.io_nongatewayroutes.yaml deploy/submariner/crds/submariner.io_routeagents.yaml $(shell find deploy/ -name "*.yaml") $(shell find config/rbac/ -name "*.yaml") $(CONTROLLER_DEEPCOPY)
-	$(GO) generate pkg/embeddedyamls/generate.go
-
-bin/%/submariner-operator: cmd/main.go $(EMBEDDED_YAMLS)
+# Generated YAMLs
+GENERATED_YAMLS := deploy/crds/submariner.io_servicediscoveries.yaml \
+                   deploy/crds/submariner.io_brokers.yaml \
+				   deploy/crds/submariner.io_submariners.yaml \
+                   deploy/submariner/crds/submariner.io_clusterglobalegressips.yaml \
+				   deploy/submariner/crds/submariner.io_clusters.yaml \
```

Comment:
These should be extracted from the Submariner constants if they’re still needed for the bundle.